src/algorithms/fardetectors/FarDetectorLinearTracking.h

```diff
@@ -0,0 +1,38 @@
+// Created by Simon Gardner
+// Subject to the terms in the LICENSE file found in the top-level directory.
+//
+
+#pragma once
+
+#include <spdlog/spdlog.h>
+
+// Event Model related classes
+#include <edm4eic/TrackParametersCollection.h>
+
+#include <extensions/jana/JChainFactoryT.h>
+#include <extensions/spdlog/SpdlogMixin.h>
+#include <spdlog/logger.h>
+#include <services/geometry/dd4hep/JDD4hep_service.h>
```

Comment:
Use `"` for all headers inside EICrecon (tree-wide).

This matters since we build EICrecon in CI while a system-wide EICrecon is already available, so using `<services/geometry/dd4hep/JDD4hep_service.h>` will find the system-wide version, while `"services/geometry/dd4hep/JDD4hep_service.h"` will find the local version which might have local changes.

We don't have a CI test for this, but really should have one.